temporal-sdk/src/main/java/io/temporal/client/WorkflowClient.java

```diff
@@ -568,6 +568,288 @@ static <A1, A2, A3, A4, A5, A6, R> WorkflowExecution start(
     return WorkflowClientInternalImpl.start(workflow, arg1, arg2, arg3, arg4, arg5, arg6);
   }
 
+  /**
+   * Executes zero argument workflow with void return type together with an update workflow request.
+   *
+   * @param workflow The only supported value is method reference to a proxy created through {@link
+   *     #newWorkflowStub(Class, WorkflowOptions)}.
+   * @param updateOperation update workflow operation
+   * @return WorkflowUpdateHandle that can be used to get the result of the update
+   */
+  static <R> WorkflowUpdateHandle<R> updateWithStart(
+      Functions.Proc workflow, @Nonnull UpdateWithStartWorkflowOperation<R> updateOperation) {
+    return updateOperation.invoke(workflow);
+  }
+
+  /**
+   * Executes one argument workflow with void return type together with an update workflow request.
+   *
+   * @param workflow The only supported value is method reference to a proxy created through {@link
+   *     #newWorkflowStub(Class, WorkflowOptions)}.
+   * @param arg1 first workflow function parameter
+   * @param updateOperation update workflow operation
+   * @return WorkflowUpdateHandle that can be used to get the result of the update
+   */
+  static <R, A1> WorkflowUpdateHandle<R> updateWithStart(
```

Comment:
Also these should be tagged `Experimental` right? I see tags in other places for Update-With-Start but not here.

Reply:
👍 addressed in https://github.com/temporalio/sdk-java/pull/2199/commits/5833afe1eafb1103d18c20573d3fc78d4c9d105f